jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/metrics/DashboardEndpoint.java

```diff
@@ -16,6 +16,7 @@
  */
 package org.jboss.aerogear.unifiedpush.rest.metrics;
 
+import com.qmino.miredot.annotations.ReturnType;
```

Comment:
:) it is impossible to analyze `javax.ws.rs.core.Response`
I'm going to refactor some API methods to return real objects instead of  `javax.ws.rs.core.Response` and use [ExceptionMapper](https://docs.oracle.com/javaee/6/api/javax/ws/rs/ext/ExceptionMapper.html) for other response codes, such 404. It will help us to minimize usage of `@ReturnType`.